File: DevExpress-Demo/CS/ControlsDemo.Wpf/DemoTesting/Fixtures.cs
using ControlsDemo.Modules;
using DevExpress.Xpf.DemoBase.DemoTesting;
using System;
using System.Linq;

namespace ControlsDemo.Tests {
    public class ControlsCheckAllDemosFixture : CheckAllDemosFixture {
        Type[] skipMemoryLeaksCheckModules = new Type[] { typeof(BreadcrumbSamplesModule) };
        protected override bool CheckMemoryLeaks(Type moduleType) {
            return !skipMemoryLeaksCheckModules.Contains(moduleType);
        }
    }
}

File: DevExpress-Demo/VB/ControlsDemo.Wpf/DemoTesting/Fixtures.vb
Imports ControlsDemo.Modules
Imports DevExpress.Xpf.DemoBase.DemoTesting
Imports System
Imports System.Linq

Namespace ControlsDemo.Tests

    Public Class ControlsCheckAllDemosFixture
        Inherits CheckAllDemosFixture

        Private skipMemoryLeaksCheckModules As Type() = New Type() {GetType(BreadcrumbSamplesModule)}

        Protected Overrides Function CheckMemoryLeaks(ByVal moduleType As Type) As Boolean
            Return Not skipMemoryLeaksCheckModules.Contains(moduleType)
        End Function
    End Class
End Namespace
